.gitignore

```diff
@@ -38,6 +38,7 @@ sd-driver
 
 #Project generated test files
 TESTS/scripts/testData
+TESTS/scripts/.venv
```

Comment:
Probably dont need to commit this

Reply:
I just want to ignore the virtual environment.